core/src/core/api/memgraph.py

```diff
@@ -0,0 +1,97 @@
+from typing import Any, Dict, List, Optional
+from neo4j import GraphDatabase
+
+
+class MemgraphClient:
```

Comment:
Not sure about naming, maybe just Memgraph?

Reply:
Changed